src/Hidden/Hidden.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import {useMedia} from '../hooks/useMedia'
+import {viewportRanges} from '../hooks/useResponsiveValue'
+
+type Viewports = 'narrow' | 'regular' | 'wide'
+
+type HiddenProps = {
+  on: Array<Viewports> | Viewports
+  children: React.ReactNode
+}
+
+export const Hidden = ({on: hiddenViewports, children}: HiddenProps) => {
```

Comment:
Would we need to worry about providing a default value for SSR in this exploration?